apps/api-extractor/src/schemas/api-extractor.schema.json

```diff
@@ -68,8 +68,17 @@
         },
 
         "reportFileName": {
-          "description": "The filename for the API report files.  It will be combined with \"reportFolder\" or \"reportTempFolder\" to produce a full file path. The file extension should be \".api.md\", and the string should not contain a path separator such as \"\\\" or \"/\".",
-          "type": "string"
+          "description": "The filename for the complete (\"untrimmed\") API report files.  It will be combined with \"reportFolder\" or \"reportTempFolder\" to produce a full file path. The file extension should be \".api.md\", and the string should not contain a path separator such as \"\\\" or \"/\".",
+          "type": ["string", "null"]
+        },
+
+        "reportVariants": {
+          "description": "The set of report variants to generate. Default: [\"complete\"]",
```

Comment:
```suggestion
          "description": "To support different approval requirements for different API release tags, multiple variants of the API report can be generated.  The \"reportVariants\" setting specifies a list of variants to be generated.  If omitted, by default only the \"complete\" variant will be generated, which includes all @alpha, @beta, and @public items.  Other possible variants are \"alpha\" (@alpha release tag only), \"beta\" (@beta only), and \"public\" (@public only).",
```

Reply:
This isn't accurate, though. As implemented, `alpha` will include anything `@alpha` or "higher" (i.e. `@alpha`, `@beta`, or `@public`). This creates a parallel between the reports and corresponding trimmed `d.ts` files.

https://github.com/microsoft/rushstack/pull/4621/files/0209edd7e0af2576df4e4bf987c2eb4ebb82c2cd#r1548764854

Will update the phrasing with your feedback though, while we settle on the semantics in the above thread.